bucket/lite-xl.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "2.0.3",
+    "description": "A lightweight text editor written in Lua, adapted from lite.",
+    "homepage": "https://lite-xl.github.io",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/lite-xl/lite-xl/releases/download/v2.0.3/lite-xl-windows-x86_64.zip",
+            "hash": "d76fb97760c06a75c433734d68a0fb5970c4a148acfb66648f0943d4a0a95471"
+        },
+        "32bit": {
+            "url": "https://github.com/lite-xl/lite-xl/releases/download/v2.0.3/lite-xl-windows-x86.zip",
+            "hash": "3f0e62793120c29c71d5b3d5f5b394fc9d897ce7cb3a99d5b54e028e14a879f0"
+        }
+    },
+    "extract_dir": "lite-xl",
+    "bin": "lite-xl.exe",
+    "shortcuts": [
+        [
+            "lite-xl.exe",
+            "Lite XL"
+        ]
+    ],
```

Comment:
Nothing to persist?

Reply:
Since I create lite-xl first, and that one put user data under `~/.config/lite-xl` that out of scoop's scope 😄 

Persist `data` now